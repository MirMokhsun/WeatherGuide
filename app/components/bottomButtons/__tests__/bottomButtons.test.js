import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import BottomButtons from '..';

describe('BottomButtons', () => {
  test('BottomButtons snapshot', () => {
    const renderer = new ShallowRenderer();
    const snapshot = renderer.render(<BottomButtons />);
    expect(snapshot).toMatchSnapshot();
  });
});
