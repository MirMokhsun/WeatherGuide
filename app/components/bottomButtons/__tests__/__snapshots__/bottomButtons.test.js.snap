// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomButtons BottomButtons snapshot 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "height": 40,
      "justifyContent": "space-between",
    }
  }
>
  <TouchableOpacity
    activeOpacity={0.2}
    onPress={[Function]}
  >
    <Icon
      allowFontScaling={false}
      color="white"
      name="sharealt"
      size={30}
    />
  </TouchableOpacity>
  <TouchableOpacity
    activeOpacity={0.2}
    onPress={[Function]}
  >
    <Icon
      allowFontScaling={false}
      color="white"
      name="staro"
      size={30}
    />
  </TouchableOpacity>
</View>
`;
